### ISSUE TITLE
STORY: As a 2diy photo experience developer, I want to run the 2diy texture demonstration on my laptop and examine existing preprocessor output, so that I can demonstrate some initial ideas for photo rendering.

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#782]
Expanding on slack discussion:

- [ ] Set up a time to get @JRegimbal's  demo code installed and running on at least one of your laptops, so you can start playing with what the 2diy can do
- [ ] Once you have the demo code running, make physical adjustments to the 2diy devices that had some glitches after Friday
- [ ] Start looking at IMAGE preprocessor output to get an idea of what is currently available for you to work with in generating photo experiences. 
- [ ] Set up a work area/safe space for storing the 2diy (@Cybernide can assist with space issues)
- [ ] Make sure you have cardkey access to the space so you can work whenever you want!
- [ ] Choose a photograph or two, and play with parameters in @JRegimbal 's demonstration, as potential mappings.

At Oct23 sprint meeting, you can demonstrate some of the textures/force feedback settings you think might be usable. Definitely not looking for anything polished, but this will demonstrate that you've got everything hooked up, understand what the preprocessors are currently giving you, and start the wheels turning on brainstorming potential experiences.

**Comment by Cybernide** [comment]
The work area has been established - the 2diy should be in the room.

**Comment by jeffbl** [comment]
Thanks @Cybernide!

Team - I know that Cyan posted some photos on slack, which is great, but also want to let you know that another source of photos is the [IMAGE test graphics repository](https://github.com/Shared-Reality-Lab/IMAGE-test-graphics).

**Comment by Cybernide** [comment]
Yes, you can feel free to use those as well - I was just giving the examples as as starting point :)

**Comment by jeffbl** [comment]
I broke out the physical adjustments to assembled 2diy's as a separate work item #785, so that just leaves looking at the preprocessor data. If you want to complete that here, that is fine, or feel free to move that specific item into your new sprint story and close this as complete.

**Comment by LukeBebee** [comment]
I added the preprocessor item to our next sprint and will close this story.
